### ISSUE TITLE
OSX (homebrew) installation results in "Error in pixReadStream" issue

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sbmbeng/tesseract-ocr#889]
```
I installed tesseract for OSX (Mountain lion - 10.8.3) using the homebrew 
installation instructions and had the "Error in pixReadStream" problem which is 
described in the FAQ section.

brew install leptonica -> version 1.69
brew install tesseract -> version 3.02.02

I agree that there was a missing image library when compiling leptonica.

The solution for me was to reinstall leptonica after installing tesseract. The 
only thing was that in the meantime before I uninstalled leptonica I installed 
imagemagick using brew and restarted so I'm not sure if this brought in the 
correct image libraries, or the path information was updated, or if purely 
reinstalling leptonica would have fixed the problem by itself.

brew install leptonica
brew install tesseract
brew install imagemagick
[restart computer]
brew uninstall leptonica
brew install leptonica

The real solution would probably be to explicitly install using brew the 
missing library (probably libtiff) using brew before starting. e.g.

brew install libtiff
brew install leptonica
brew install tesseract



```

Original issue reported on code.google.com by `ja...@somethingwild.co.nz` on 7 Apr 2013 at 11:29
- Merged into: #887
